lib/mercury/wire_serializer.rb

```diff
@@ -23,17 +23,15 @@ def write_json(obj)
     end
 
     def read_json(bytes)
-      begin
-        Oj.compat_load(bytes, oj_options)
-      rescue Oj::ParseError => e
-        bytes
-      end
+      Oj.compat_load(bytes, oj_options)
+    rescue Oj::ParseError => e
+      bytes
```

Comment:
I generally prefer this and have been using it more often than not - it jumps out at me faster and it is more succinct